provisioning/apps-proxy/kubernetes/templates/cloud/azure/service.yaml

```diff
@@ -5,13 +5,13 @@ metadata:
   name: apps-proxy
   namespace: $NAMESPACE
   labels:
-    app: apps-proxy
+    app: $APP
   annotations:
     service.beta.kubernetes.io/azure-load-balancer-internal: "true"
 spec:
   type: LoadBalancer
   selector:
-    app: apps-proxy
```

Comment:
je nutný tohle parametrizovat? nemohl by ten nginx redirect mít stejnej label? nebo bys to považoval za antipattern?

Reply:
zeptal jsem jsem AI:
```
Best Practice:

If you need two separate Deployments but want different traffic handling for each, it’s better to:

    Use different label sets for the pods in each Deployment (e.g., app: app-proxy-1 and app: app-proxy-2).
    Create separate Services with appropriate selectors to route traffic to the pods of each Deployment separately.

But, if you intend for both Deployments' pods to handle the same traffic, the same selector labels will work fine. Just keep in mind the implications for traffic routing, updates, and scaling.
```